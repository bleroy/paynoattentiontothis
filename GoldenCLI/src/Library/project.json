{
  "version": "1.0.0-*",
  "buildOptions": {
    "emitEntryPoint": true
  },
  "dependencies": {
    "NETStandard.Library": "1.5.0-rc2-24027",
    "Microsoft.NETCore.Portable.Compatibility": "1.0.1-rc2-24027"
  },
  "frameworks": {
    "netstandard1.5": {
      "imports": ["portable-net40+sl5+win8+wp8+wpa81", "dotnet"]
    }
  }
}
