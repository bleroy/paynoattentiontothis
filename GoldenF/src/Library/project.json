{
  "version": "1.0.0-*",
  "buildOptions": {
    "compilerName": "fsc",
    "compile": {
      "includeFiles": [
        "Thing.fs"
      ]
    }
  },
  "dependencies": {
    "Microsoft.FSharp.Core.netcore": "1.0.0-alpha-160316",
    "Microsoft.NETCore.App": {
      "type": "platform",
      "version": "1.0.0-rc2-3002702"
    },
    "System.Runtime.Serialization.Primitives": "4.1.1-rc2-24027",
    "Microsoft.NETCore.Portable.Compatibility": "1.0.1-rc2-24027",
    "Newtonsoft.Json": "8.0.3"
  },
  "tools": {
    "dotnet-compile-fsc": {
      "version": "1.0.0-*",
      "imports": [
        "dnxcore50",
        "portable-net45+win81",
        "netstandard1.3"
      ]
    }
  },
  "frameworks": {
    "netstandard1.5": {
      "imports": [
        "portable-net40+sl5+win8+wp8+wpa81",
        "dnxcore50",
        "dotnet"
      ]
    }
  }
}
