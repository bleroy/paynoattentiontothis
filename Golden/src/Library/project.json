{
  "version": "1.0.0-*",

  "dependencies": {
    "Microsoft.NETCore.Portable.Compatibility": "1.0.1-rc2-24027",
    "NETStandard.Library": "1.5.0-rc2-24027",
    "Newtonsoft.Json": "8.0.3",
    "System.Runtime.Serialization.Primitives": "4.1.1-rc2-24027"
  },
  "frameworks": {
    "netstandard1.5": {
      "imports": [ "portable-net40+sl5+win8+wp8+wpa81", "dotnet" ]
    }
  }
}
